frontend/src/ui/Access.tsx

```diff
@@ -525,44 +564,36 @@ const Warning: React.FC<WarningProps> = ({ tooltip, info }) => (
     </WithTooltip>
 );
 
-const UnchangeableAllActions = () => {
+const UnchangeableAllActions: React.FC<{ item?: Entry}> = ({ item }) => {
```

Comment:
Mh ok. I wonder why you are passing the whole item. Why not just a `PermissionLevel`?

Reply:
fair, though that needs type assertion at some point, like `<UnchangeableAllActions permission={item.label as PermissionLevel} />`.
In theory I would like to give the `Entry` and `Acl` types also `actions: Set<PermissionLevel>` (instead of `Set<string>`) to get around the assertion. But that in turn needs backend adjustments... which I think should be possible? 🤷